setup_wordpress.sh

```diff
@@ -1,3 +1,103 @@
 #!/bin/bash
+set -e
+echo '################## LEMP Stack Installation ####################'
+cd ~
+echo "------------------ Updating APT ------------------"
+echo
+sudo apt -y update && sudo apt -y upgrade
 
-echo 'This script should install and setup Wordpress'
+echo "------------------ Installing Nginx ------------------"
+sudo apt -y install nginx
+
+echo "------------------ Configuring Firewall ------------------"
+sudo ufw enable
+sudo ufw allow 'Nginx Full'
+
+echo "------------------ Installing MYSQL ------------------"
+sudo apt -y install mysql-server
+sudo mysql_secure_installation
```

Comment:
on Vagrant it's stops on this step.

Reply:
Yes, I realised that. I'm trying to fix it.